src/c++/perf_analyzer/genai-perf/genai_perf/utils.py

```diff
@@ -41,6 +41,11 @@ def encode_image(img: Image, format: str):
     import base64
     from io import BytesIO
 
+    # JPEG does not support P or RGBA mode (commonly used for PNG) so it needs
```

Comment:
is this the right place for this method?
is it used throughout the code base or just 1 file?

Reply:
works for me. i just wanted to check